conf/constlables.go

```diff
@@ -0,0 +1,34 @@
+package conf
+
+import (
+	"github.com/megaease/easeprobe/probe"
+)
+
+var constLabels = make(map[string]bool)
```

Comment:
why it is a global variable? I think we can make it a local variable by refactor the code below, this could make the code more maintainable and avoid potential bugs.

Reply:
this was a digress of an original design. updated.